assets/samples/cloudNativeFullStack.yaml

```diff
@@ -99,32 +105,46 @@ spec:
       #     cpu: 300m
       #     memory: 1.5Gi
 
-      # Optional: disables automatic restarts of OneAgent pods in case a new version is available. True
-      # by default.
+      # Optional: Adds arguments to add to the OneAgent installer
+      # For a list of available options, see https://www.dynatrace.com/support/help/shortlink/linux-custom-installation
+      # For a list of the limitations for OneAgents in Docker, see https://www.dynatrace.com/support/help/shortlink/oneagent-docker#limitations
+      #
+      # args: []
+
+      # Optional: Adds additional environment variables to OneAgent pods
+      #
+      # env: []
+
+      # Optional: Disables automatic restarts of OneAgent pods in case a new version is available
```

Comment:
For me this sentence is a bit ambiguous. A pod restart is an implication of an update, but the flag is about the update itself, may I humble suggest something along the lines of:

"Enables automatic OneAgent updates. Disable them to avoid unwanted automatic OneAgent pod restarts caused by automatic updates."